### ISSUE TITLE
LodgeitUpload: lexer[1][0] IndexError: tuple index out of range

### ISSUE THREAD

**Issue by Duckle29** [bepasty/bepasty-server#258]
Hi there.

In a personal server re-vamp, I decided I wanted to also have pasting available self-hosted, and I found this awesome project.
As I deploy my VPS mostly through ansible managed docker images, I wanted to do so for bepasty too.

In order to do this I looked around, and found a 5 year old image. I decided to try and move this image to the latest alpine, and have a scheduled github action to regularly(monthly) rebuild the image (to follow the latest pypi build of bepasty-server)

I however have run into some issues getting it running, and have gotten stuck at the following error when uWSGI tries to start:

```traceback
Traceback (most recent call last):
  File "wsgi.py", line 4, in <module>
    from bepasty.app import create_app
  File "/usr/lib/python3.8/site-packages/bepasty/app.py", line 14, in <module>
    from .apis import blueprint as blueprint_apis
  File "/usr/lib/python3.8/site-packages/bepasty/apis/__init__.py", line 3, in <module>
    from .lodgeit import LodgeitUpload
  File "/usr/lib/python3.8/site-packages/bepasty/apis/lodgeit.py", line 15, in <module>
    class LodgeitUpload(MethodView):
  File "/usr/lib/python3.8/site-packages/bepasty/apis/lodgeit.py", line 25, in LodgeitUpload
    name = lexer[1][0]
IndexError: tuple index out of range
unable to load app 0 (mountpoint='') (callable not found or import error)
```

Any pointers in the right direction would be greatly appreciated :)

_edit: docker image source is here https://github.com/Duckle29/bepasty-docker_

**Comment by ThomasWaldmann** [comment]
I just merged a PR yesterday into master that fixes this.

Guess it is related to some recent change in pygments, so either using our master branch code or an older pygments release would both likely fix that.

**Comment by ThomasWaldmann** [comment]
Try: `pygments<2.7.3`

**Comment by Duckle29** [comment]
That indeed fixed it for pip installs :)
I've tagged that as `stable` on my docker repo, and will try installing from git on the `latest` tag. Any way to use `[magic]` with the editable pip install?

**Comment by Duckle29** [comment]
And installing the latest from git also works! Awesome, now I have exactly the pastebin I want with the deployment method I want :D

Test paste here https://paste.mikkel.cc/shzo9Dzo

Thanks for the help, and I'm happy to see that this is still actively maintained. Lots of other bins I looked at weren't.

**Comment by foosinn** [comment]
Any chance of a bugfix or new release? `pip install bepasty` without further options seems currently broken.

**Comment by ThomasWaldmann** [comment]
@foosinn yes, i'll make a release soon.

meanwhile, you can just install the older pygments to make the last release work again, see above.

**Comment by reivilibre** [comment]
I just also ran into this. `pygments<2.7.3` as an extra requirement seems to have indeed worked around the issue.

**Comment by mweinelt** [comment]
Downgrading pygments to a version before 2.7.4 should not be a recommended solution. 

See https://github.com/pygments/pygments/issues/1625 (CVE-2021-20270)

**Comment by ThomasWaldmann** [comment]
@mweinelt just released 1.0.0. \o/
